sensor_msgs/Image img
PoseXYZ pose # units: m
float64[8] imgCornersXY # XY pairs order: bottom-left, bottom-right, top-right, top-left; units: m